### DISCUSSION TITLE
Timeline w.r.t. exec_run

### DISCUSSION THREAD

**Discussion by drgsr** [containers/podman-py#107] [Q&A] [answered]
Hello,
in my project, I would require/need/use the container.exec_run feature.
As I see from the code, the API is defined, but there is no implementation 

https://github.com/containers/podman-py/blob/main/podman/domain/containers.py
Line 176: raise NotImplementedError()

What is the intended timeline for this feature?
Thx in advance,
Günter

**Comment by JacobCallahan** [comment, marked as answer]
Hi @drgsr this is something that I decided to take on. You can follow the progress in #126. Testing is always appreciated :)
